AppCenterAnalytics/AppCenterAnalytics/TransmissionTarget/MSAnalyticsTransmissionTarget.m

```diff
@@ -21,6 +26,9 @@ - (instancetype)initWithTransmissionTargetToken:(NSString *)token
     if (![self isImmediateParent]) {
       [MS_USER_DEFAULTS setObject:@(NO) forKey:self.isEnabledKey];
     }
+
+    // Add property configurator to the channel group as a delegate.
+    [_channelGroup addDelegate:_propertyConfigurator];
```

Comment:
I approved but had a question about this that Jae answered in slack.
I don't like how the TransmissionTarget object holds onto the channel group reference and modifies it (it adds a new delegate). I understand that the PropertyConfigurator is specific to a TransmissionTarget but all other options that come to mind feel dirty, too.

Reply:
Yes. This is not the best but better than all other possible options.